src/backend/data/post.js

```diff
@@ -148,7 +149,16 @@ class Post {
       article.date = article.pubdate;
     }
 
+    // All the Youtube feed return an array off html
+    if (determinePostType(article.link) === 'video' && Array.isArray(article.content)) {
```

Comment:
I think we should check if `Array.isArray(article.content)` only and drop the video check.